modules/tensor_mechanics/doc/content/documentation/systems/UserObjects/GeneralizedPlaneStrainUserObject.md

```diff
@@ -21,6 +21,12 @@ and the corresponding diagonal jacobian is
 	K_{zz} = \frac{\partial R_{zz}}{\partial \epsilon_{zz}} = \int_{A}{\frac{\partial \sigma_{zz}}{\partial \epsilon_{zz}}dA} = \int_{A}{C_{2222}dA}
 \end{equation}
 
+The reference residual value used by [GeneralizedPlaneStrainReferenceResidual](/systems/AuxScalarKernels/GeneralizedPlaneStrainReferenceResidual.md) is computed as
```

Comment:
remove systems from the path should get the docs to build successfully, I believe